functions/Copy-DbaDbTableData.ps1

```diff
@@ -38,17 +38,22 @@ function Copy-DbaDbTableData {
         The database to copy the table to. If not specified, it is assumed to be the same of Database
 
     .PARAMETER Table
-        Define a specific table you would like to use as source. You can specify a three-part name like db.sch.tbl.
+        Specify a table to use as a source. You can specify a two-part name or a three-part name such as db.sch.tbl.
         If the object has special characters please wrap them in square brackets [ ].
-        This dbo.First.Table will try to find table named 'Table' on schema 'First' and database 'dbo'.
-        The correct way to find table named 'First.Table' on schema 'dbo' is passing dbo.[First.Table]
+        Example: [SampleDB].[First].[Table] will try to find the table named 'Table' in the schema 'First' and the database 'SampleDB'.
+
+    .PARAMETER View
+        Specify a view to use as a source. You can specify a two-part name or a three-part name such as db.sch.vw.
+        Note: Only one of -View or -Table may be specified during command invocation.
+        If the object has special characters please wrap them in square brackets [ ].
+        Example: [SampleDB].[First].[View] will try to find the view named 'View' in the schema 'First' and the database 'SampleDB'.
```

Comment:
I noticed known of our examples for this command show the 3-part name being used. 

I think it will serve better having this as a formal example. Along with the same for the table parameter.

Reply:
Yes, I agree